posix.mak

```diff
@@ -139,6 +142,11 @@ endif
 ifdef ENABLE_COVERAGE
 DFLAGS  += -cov
 endif
+ifneq (,$(TZ_DATABASE_DIR))
+$(file > TZDatabaseDirFile, ${TZ_DATABASE_DIR})
+DFLAGS += -version=TZDatabaseDir -J.
```

Comment:
Thanks.
Final remark: Advantage of using `$(ROOT)/TZDatabaseDirFile` here would be that you will have a clean `git status`.

Reply:
Well with `/tmp` you run into the problem of some people not having this directory as `/tmp` isn't available on all OS nor all Linux distributions, but we will rework our Makefile soon anyways and as long as you pass `TZ_DATABASE_DIR` as variable, everything will be working.

Reply:
What kind of rework?

Reply:
The goal is to get rid of the ugly `win{32,64}.mak` files and unite these into one Makefile.